### PR TITLE
OBSDOCS-890: Updates for http input receiver docs

### PR DESCRIPTION
**Pull request by abrennan89**
Version(s):
4.12+

Issue:
https://issues.redhat.com/browse/OBSDOCS-890

Link to docs preview:
https://73567--ocpdocs-pr.netlify.app/openshift-enterprise/latest/logging/log_collection_forwarding/cluster-logging-collector#cluster-logging-collector-input-receivers

QE review:
- [x] QE has approved this change.

Additional information:
Follow up from syslog bug https://issues.redhat.com/browse/OBSDOCS-890 to also provide these updates for the 4.12+ HTTP docs

### CODE REVIEW

**Review by jcantrill — COMMENTED** [review]
/lgtm

File: modules/log-collector-http-server.adoc [hunk 1]
```diff
@@ -37,16 +39,47 @@ spec:
   pipelines: # <5>
     - name: http-pipeline
       inputRefs:
-        - http-receiver
+        - infrastructure
```
Comment (anpingli) [inline modules/log-collector-http-server.adoc:42]: http-receiver should be kept.

File: modules/log-collector-http-server.adoc [hunk 2]
```diff
@@ -37,16 +39,47 @@ spec:
...
+  inputs:
+    - name: http-receiver # <1>
+      receiver:
+        type: http # <2>
+        http:
+          format: kubeAPIAudit # <3>
+          port: 8443 # <4>
+  pipelines: # <5>
+  - inputRefs:
+    - infrastructure
```
Comment (anpingli) [inline modules/log-collector-http-server.adoc:71]: http-receiver should be kept.

File: modules/log-collector-http-server.adoc [hunk 3]
```diff
@@ -45,8 +47,39 @@ spec:
...
+          port: 8443 # <4>
+  pipelines: # <5>
+  - inputRefs:
+    - http-receiver
+    name: http-pipeline
+# ...
+----
+<1> Specify a name for your input receiver.
+<2> Specify the input receiver type as `http`.
+<3> Currently, only the the `kube-apiserver` webhook format is supported for `http` input receivers.
```
Comment (GroceryBoyJr) [inline modules/log-collector-http-server.adoc:77]:
```suggestion
<3> Currently, only the `kube-apiserver` webhook format is supported for `http` input receivers.
```

**Review by GroceryBoyJr — COMMENTED** [review]
Only one tiny nit, otherwise LGTM.

File: modules/log-collector-http-server.adoc (hunk 3 above)
Comment (abrennan89) [reply modules/log-collector-http-server.adoc:77]: Thank you, great catch!

**Pull request merged**